Library/Homebrew/bintray.rb

```diff
@@ -106,10 +112,18 @@ def upload_bottle_json(json_files, publish_package: false)
     formula_packaged = {}
 
     bottles_hash.each do |formula_name, bottle_hash|
+      formula = Formula[formula_name]
       version = ERB::Util.url_encode(bottle_hash["formula"]["pkg_version"])
       bintray_package = bottle_hash["bintray"]["package"]
       bintray_repo = bottle_hash["bintray"]["repository"]
 
+      stable_urls = [formula.stable.url] + formula.stable.mirrors
+      stable_urls.grep(%r{^https://dl.bintray.com/homebrew/mirror/}) do |mirror_url|
+        next if stable_mirrored?(mirror_url)
+
+        safe_system "#{HOMEBREW_PREFIX}/bin/brew", "mirror", formula_name
```

Comment:
Could maybe call the mirroring logic directly rather than shelling out?

Reply:
Good point, I pulled the logic from `dev-cmd/mirror.rb` to a new method in `bintray.rb`.